decidim-core/spec/models/decidim/user_group_spec.rb

```diff
@@ -72,6 +72,11 @@ module Decidim
         it { is_expected.not_to be_valid }
       end
 
+      # This test fails because the test validates  ActionDispatch::Http::UploadedFile
+      # but in this case the model receives an ActiveStorage::Attached and if we try
+      # to analyze the blob a ActiveStorage::FileNotFoundError Exception is
+      # raised. The code commented there works but it seem too tricky
+      # https://github.com/rails/rails/issues/36994#issuecomment-523426645
```

Comment:
This test is fixed with 6487b39 and providing a blob for the attribute (f61cfc3)